src/components/Article.jsx

```diff
@@ -1 +1,15 @@
-// export function Article() {}
+import React from 'react';
+
+export function Article({
+  tittle,
```

Comment:
Correctly write this way 😉 
```suggestion
  title,
```